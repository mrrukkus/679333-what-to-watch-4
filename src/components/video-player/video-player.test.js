import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "../video-player/video-player.jsx";

const src = `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`;

it(`Video-player renders correctly`, () => {
  const player = renderer
    .create(<VideoPlayer
      isPlaying={false}
      muted={true}
      src={src}
      poster={`some poster src`}
      playAndPauseHandler={() => {}}
    ></VideoPlayer>, {
      createNodeMock: () => {
        return {};
      }
    }).toJSON();

  expect(player).toMatchSnapshot();
});
