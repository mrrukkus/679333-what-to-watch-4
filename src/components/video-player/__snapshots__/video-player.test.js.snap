// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video-player renders correctly 1`] = `
<video
  height="175"
  width="280"
/>
`;
